CONTRIBUTING.md

```diff
@@ -0,0 +1,276 @@
+# Contributing
+
+Craft Store has a community from all over the world, and we welcome all contributions.
+
+All contributors should become familiar with this guide. It outlines the expectations
+and practices for participating in the project.
+
+## Review the project expectations
+
+The Starcraft team at Canonical sets the direction and priorities of Craft Store. They
+take responsibility for its stewardship and health.
+
+Before you start work on Craft Store, there are three documents for you to digest.
+
+### Ubuntu Code of Conduct
+
+Projects governed by Canonical expect good conduct and excellence from every member. The
+specific principles are laid out in the [Ubuntu Code of
+Conduct](https://ubuntu.com/community/ethos/code-of-conduct).
+
+### Canonical Contributor License Agreement
+
+As a contributor, you retain your copyright and attribution rights, provided you sign
+the [Canonical Contributor License Agreement](http://www.ubuntu.com/legal/contributors).
+Before committing any code, review its terms. If you agree and sign it, your code can be
+incorporated into the repository.
+
+### Open source license
+
+Craft Store is licensed under [LGPL-3.0](LICENSE).
+
+## Report an issue or open a request
+
+If you find a bug or feature gap in Craft Store, look for it in the [project's GitHub
+issues](https://github.com/canonical/craft-store/issues) first. If you have fresh input,
+add your voice to the issue.
+
+If the bug or feature doesn't have an issue, we invite you to [open
+one](https://github.com/canonical/craft-store/issues/new/choose).
+
+## Set up for development
+
+Craft Store uses a forking, feature-based workflow. Most work on Craft Store occurs on
+people's local systems, and is heavily terminal-dependent. Remote testing and building
+is provided on GitHub for continuous integration and delivery.
+
+Start by [creating a personal fork](https://github.com/canonical/craft-store/fork) of
+the repository on GitHub.
```

Comment:
Since we're telling them to base on a hotfix branch, we should tell them here to deselect "copy the `main` branch only"

Reply:
Good point!

As an alternative, what about adding `git fetch --tags --all` [here](https://github.com/canonical/craft-store/blob/34d7c34f61b2a9cd0036627bd4b82f8911072160/CONTRIBUTING.md?plain=1#L113)? That way they'll have all the current branches.

Reply:
I think it's fine to leave it the same below since they'll be wanting to sync their repository before starting work if they already have a fork. Maybe here we should point out that if they already have a personal fork they should sync it?

Reply:
Getting the bounty info out is the highest priority right now, so I'm going to merge as-is. This question affects all the other repos, too, so we'll revisit in CRAFT-4627.